action_server/src/action_server/actions/__init__.py

```diff
@@ -2,17 +2,18 @@
 
 from answer_question import AnswerQuestion
 from arm_goal import ArmGoal
-from bring import Bring
```

Comment:
Why no more Bring?

Reply:
Because we now have "hand over" and "place" actions. Both those actions trigger the appropriate actions to make sure there is something to hand over or place, respectively. This means `bring me a coke` is the same thing as `hand me a coke` and will result in the desired behavior.